### DISCUSSION TITLE
undici pool  clients problem

### DISCUSSION THREAD

**Discussion by 1119302165** [nodejs/undici#1561] [Q&A]
Pool Params
[Parameter: PoolOptions](https://undici.nodejs.org/#/docs/api/Pool?id=parameter-pooloptions)
Extends: [ClientOptions](https://undici.nodejs.org/#/docs/api/Client?id=parameter-clientoptions)

factory (origin: URL, opts: Object) => Dispatcher - Default: (origin, opts) => new Client(origin, opts)
connections number | null (optional) - Default: null - The number of Client instances to create. When set to null, the Pool instance will create an unlimited amount of Client instances.

if i set connections = 1000

Whether the client can send information to the server every second to keep connected？

**Comment by mcollina** [comment]
Could you clarify your question?
